src/RequestHandler.php

```diff
@@ -245,12 +261,26 @@ private function toServerResponse(
             }
         }
 
+        if ($symfonyResponse instanceof BinaryFileResponse) {
```

Comment:
This is inconsistent, right? I mean, if that response can be another thing but a Symfony Response, then is not `$symfonyResponse` but `$response`.

Reply:
`BinaryFileResponse` extends the Symfony response in the same way as `StreamedResponse` does. 

See https://github.com/symfony/http-foundation/blob/master/StreamedResponse.php and https://github.com/symfony/http-foundation/blob/master/BinaryFileResponse.php

So both objects pass the method type hint but need different handling within the method to benefit from the streaming capabilities.

Reply:
You're right ^^